jwql/instrument_monitors/nircam_monitors/claw_monitor.py

```diff
@@ -193,7 +193,7 @@ def make_background_plots(self, plot_type='bkg'):
                     plot_data = df['stddev'].values
                 if plot_type == 'model':
                     total_bkg = [1. for x in df['median'].values]
```

Comment:
I think this line isn't needed anymore, but it looks harmless:
`total_bkg = [1. for x in df['median'].values]`

Reply:
You're right, good catch, fixed.